src/onegov/org/models/traitinfo.py

```diff
@@ -124,7 +134,7 @@ def get_form_class(
     def get_editbar_links(
         self,
         request: 'OrgRequest'
-    ) -> 'Sequence[Link | LinkGroup]':
+    ) -> 'Sequence[Link | LinkGroup | iFrameLink]':
```

Comment:
You shouldn't need to change this union, since `iFrameLink` is a subclass of `Link`

Reply:
I see the problem, it inherits from `BaseLink`. So you should change this to `BaseLink | BaseLinkGroup` or change `IFrameLink` and `QrCodeLink` to inherit from `Link` instead of `BaseLink`. I'm not entirely sure why we use different link classes in Org to begin with, part of it seems to be `access`, but we could make this work in the base class. I'm also not sure why we're using templates in one place and a markup builder in the other. Ideally we'd move all of that logic back into the base class.

Reply:
We're also not very consistent about which `Link` class we use, sometimes we use the base class and sometimes the Org class, regardless of whether the module is using an OrgApp or not.